inference-engine/tests/functional/plugin/gna/shared_tests_instances/subgraph_tests/concat_multi_input.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "subgraph_tests/concat_multi_input.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+std::vector<std::vector<std::vector<size_t>>> inShapes = {
+    {{1, 8}, {1, 8}},
+    {{1, 3}, {1, 3}, {1, 3}},
+    {{1, 16}, {1, 16}, {1, 16}},
+    {{1, 16}, {1, 16}, {1, 16}, {1, 16}},
+    {{1, 32}, {1, 32}, {1, 32}, {1, 32}},
+    {{1, 16}, {1, 32}, {1, 16}, {1, 32}, {1, 16}, {1, 32}},
+};
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+    InferenceEngine::Precision::FP32,
+    InferenceEngine::Precision::FP16,
+};
+
+std::map<std::string, std::string> additional_config = {
+    {"GNA_COMPACT_MODE", "NO"},
+    {"GNA_SCALE_FACTOR_0", "2048"},
```

Comment:
for integer mode computation on gna you need to you config keys:

    {"GNA_DEVICE_MODE", "GNA_SW_EXACT"},
    {InferenceEngine::GNAConfigParams::KEY_GNA_PRECISION, "I16"},

Reply:
done, added

Reply:
great - how about GNA_DEVICE_MODE? is that set somewhere else - not here?